tools/src/main/java/com/nedap/archie/archetypevalidator/validations/SpecializedDefinitionValidation.java

```diff
@@ -153,6 +160,54 @@ private void validateConformsTo(CObject cObject, CObject parentCObject) {
 
     }
 
+    private ConformanceCheckResult childNodesConformToParent(CObject childCObject, CObject parentCObject, ConformanceCheckResult conformanceCheckResult) {
+        if (!conformanceCheckResult.getErrorType().equals(ErrorType.VSONCO)) {
+            return conformanceCheckResult;
+        }
+
+        // Edge case: if sum of all occurrences of all redefined object nodes conforms to occurrences of parent object node
+        // it is valid to exclude the parent object node
+        MultiplicityInterval parentNodeOccurrences = parentCObject.effectiveOccurrences(combinedModels::referenceModelPropMultiplicity);
+        List<CObject> allRedefinedNodes = childCObject.getParent().getChildren().stream().filter(
+                child -> child.nodeIdConformsTo(parentCObject)
+                        && !child.getNodeId().equals(parentCObject.getNodeId())
+        ).collect(Collectors.toList()); // All child nodes in the child archetype
+        allRedefinedNodes.addAll(((CAttribute) flatParent.itemAtPath(parentCObject.getParent().getPath())).getChildren().stream().filter(
+                child -> child.nodeIdConformsTo(parentCObject)
+                        && !child.getNodeId().equals(parentCObject.getNodeId())
+        ).collect(Collectors.toList())); // All child nodes in the flattened parent archetype
+        MultiplicityInterval allRedefinedNodeOccurrencesSummed = new MultiplicityInterval(0, 0);
+        for (CObject childNode : allRedefinedNodes) {
+            if (allRedefinedNodeOccurrencesSummed.isOpen()) {
+                break;
+            }
+            MultiplicityInterval redefinedOccurrences = childNode.effectiveOccurrences(combinedModels::referenceModelPropMultiplicity);
+            if (!allRedefinedNodeOccurrencesSummed.isLowerUnbounded()) {
```

Comment:
In my understanding if the Interval lower or upper is unbounded, the boolean is true and the corresponding value (lower / upper) is null. Is it possible that there is a combination of:
* isLowerUnbounded() is false and getLower() equals null or 
* isLowerUnbounded() is true and getLower() equals not null
(same for upper bound)

Reply:
isLowerUnbounded() false and getLower() == null : this is invalid data, and should fail validation. I think the validation for that is still missing, but it cannot be created from ADL

isLowerUnbounded True and getLower equals not null just means getLower() can be treated as if it was null, so, lower unbounded.